src/app/cli/src/output/output_write.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use internal_error::*;
+
+use crate::OutputConfig;
+
+///////////////////////////////////////////////////////////////////////////////
+
+pub fn write_output<T: serde::Serialize>(
+    value: T,
+    output_config: &OutputConfig,
+    output_format: Option<impl AsRef<str>>,
+) -> Result<(), InternalError> {
```

Comment:
It's nice you've isolated this, but it isn't used anywhere except previous place.
Perhaps, your idea was to re-use it in `kamu system diagnose`.

Reply:
Yeah, In first version it was shared for both `info` and `diagnose`. Rollback these changes